### PR TITLE
[FIX] Added missing license classifiers to templates

### PR DESCRIPTION
**Pull request by Imipenem**

### CODE REVIEW

File: cookietemple/create/templates/gui/gui_java/cookiecutter.json [hunk 1]
```diff
@@ -8,5 +8,7 @@
   "project_slug": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}",
   "project_short_description": "GUI Java contains all the boilerplate you need to create a standalone JavaFX application.",
   "version": "1.0.0",
-  "license": ["MIT", "BSD", "ISC", "Apache2.0", "GNUv3", "Not open source"]
+    "license": ["MIT", "BSD", "ISC", "Apache2.0", "GNUv3", "Not open source",
```
Comment (Zethson) [inline cookietemple/create/templates/gui/gui_java/cookiecutter.json:11]: looks differently intended?

File: cookietemple/create/templates/lib/lib_cpp/cookiecutter.json [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
-	"version": "0.1.0", 
-	"license": [ "MIT", "BSD", "ISC", "Apache2.0", "GNUv3", "Not open source" ]
+    "full_name": "Homer Simpson",
+	"email": "homer.simpson@example.com",
+	"github_username": "homersimpson",
+	"project_name": "Modern C++ Template",
+	"project_slug": "modern-cpp-template",
+	"project_short_description": "A template for modern C++ projects, using CMake.",
+	"version": "0.1.0",
+    "license": ["MIT", "BSD", "ISC", "Apache2.0", "GNUv3", "Not open source",
```
Comment (Zethson) [inline cookietemple/create/templates/lib/lib_cpp/cookiecutter.json:9]: intendation?

File: cookietemple/create/templates/lib/lib_cpp/cookiecutter.json [hunk 3]
```diff
@@ -1,11 +1,13 @@
 {
-       	"full_name": "Homer Simpson", 
-	"email": "homer.simpson@example.com", 
-	"github_username": "homersimpson", 
-	"project_name": "Modern C++ Template", 
-	"project_slug": "modern-cpp-template", 
-	"project_short_description": "A template for modern C++ projects, using CMake.", 
-	"version": "0.1.0", 
-	"license": [ "MIT", "BSD", "ISC", "Apache2.0", "GNUv3", "Not open source" ]
+    "full_name": "Homer Simpson",
```
Comment (Zethson) [inline cookietemple/create/templates/lib/lib_cpp/cookiecutter.json:2]: intendation?

**Comment by Zethson** [comment]
Could you please create a cli-python e.g. with Affero and twine check it?

**Comment by Imipenem** [comment]
> Could you please create a cli-python e.g. with Affero and twine check it?



> Could you please create a cli-python e.g. with Affero and twine check it?

Tested; it works

**Comment by Zethson** [comment]
LGTM

**Pull request merged**